infra-templates/secrets-bridge-v2/0/rancher-compose.yml

```diff
@@ -0,0 +1,39 @@
+.catalog:
+    name: "Rancher Secrets Bridge V2"
+    version: v0.0.1
```

Comment:
can we have this match the version of the `rancher/secrets-bridge-v2` image? so make it v`0.1.0`?